Core/StartupArgManager.cs

```diff
@@ -37,5 +36,27 @@ public static string[] MergeStartupArgs(IEnumerable<string> cmdArgs)
                 return startupArgs.ToArray();
             }
         }
+
+        public static void MigrateArgsFile()
+        {
+            try
+            {
+                string OldFileLocation = "laargs.txt";
+
+                if (!File.Exists(StartupArgsPath)) { return; }
+                if (string.IsNullOrEmpty(File.ReadAllText(OldFileLocation)) || string.IsNullOrWhiteSpace(File.ReadAllText(OldFileLocation)))
```

Comment:
1. You read the entire file twice and a third time a couple of lines below. Just read it once and save in a variable.
2. There is no need to call both IsNullOrEmpty and IsNullOrWhiteSpace. If IsNullOrEmpty is true then IsNullOrWhiteSpace must be true as well.